src/endpoints/seed/index.ts

```diff
@@ -838,40 +821,59 @@ export const seed = async ({
     ),
   )
 
-  payload.logger.info(`— Seeding globals...`)
-
-  await Promise.all([
-    payload.updateGlobal({
-      slug: 'footer',
-      data: {
-        navItems: [
-          {
-            link: {
-              type: 'custom',
-              label: 'Admin',
-              url: '/admin',
-            },
-          },
-          {
-            link: {
-              type: 'custom',
-              label: 'Source Code',
-              newTab: true,
-              url: 'https://github.com/payloadcms/payload/tree/main/templates/website',
-            },
-          },
-          {
-            link: {
-              type: 'custom',
-              label: 'Payload',
-              newTab: true,
-              url: 'https://payloadcms.com/',
-            },
-          },
-        ],
+  payload.logger.info(`— Seeding footers...`)
+  const footerData: Record<Tenant['slug'], {}> = {
+    nwac: {
+      address: '249 Main Ave. S, Suite 107-366\nNorth Bend, WA 98045\n(206) 909-0203',
+      email: 'info@nwac.us',
+      socialMedia: {
+        instagram: 'https://www.instagram.com/nwacus',
+        facebook: 'https://www.facebook.com/NWACUS/',
+        twitter: 'https://x.com/nwacus',
+        linkedin: 'https://www.linkedin.com/company/nw-avalanche-center',
+        youtube: 'https://www.youtube.com/channel/UCXKN3Cu9rnnkukkiUUgjzFQ',
       },
-    }),
-  ])
+      contentHash: null,
```

Comment:
nit: you could omit these here

Reply:
I got errors from `upsert.ts` when I did. I think I need it since it's a collection now?

Reply:
Sorry, I meant that you could omit explicitly setting `contentHash: null` in this data, not that you could omit having the field on the collection itself. No biggie either way.